### PR TITLE
update bob_token and alice_token in markdown

### PR DESCRIPTION
**Pull request by rokkiter**
I passed https://play.openpolicyagent.org/p/aA8dWcuc73
This URL updated the token to solve the problem of expired decode_verify verification

### CODE REVIEW

File: docs/content/http-api-authorization.md [hunk 1]
```diff
@@ -303,8 +303,8 @@ opa build example-jwt.rego
 For convenience, we'll want to store user tokens in environment variables (they're really long).
 
 ```shell
-export ALICE_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoiYWxpY2UiLCJhenAiOiJhbGljZSIsInN1Ym9yZGluYXRlcyI6W10sImhyIjpmYWxzZX0.rz3jTY033z-NrKfwrK89_dcLF7TN4gwCMj-fVBDyLoM"
-export BOB_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoiYm9iIiwiYXpwIjoiYm9iIiwic3Vib3JkaW5hdGVzIjpbImFsaWNlIl0sImhyIjpmYWxzZX0.n_lXN4H8UXGA_fXTbgWRx8b40GXpAGQHWluiYVI9qf0"
+export ALICE_TOKEN="eyJhbGciOiAiSFMyNTYiLCAidHlwIjogIkpXVCJ9.eyJleHAiOiAyMjQxMDgxNTM5LCAibmJmIjogMTUxNDg1MTEzOSwgInJvbGUiOiAiZ3Vlc3QiLCAic3ViIjogIllXeHBZMlU9In0.Uk5hgUqMuUfDLvBLnlXMD0-X53aM_Hlziqg3vhOsCc8"
+export BOB_TOKEN="eyJhbGciOiAiSFMyNTYiLCAidHlwIjogIkpXVCJ9.eyJleHAiOiAyMjQxMDgxNTM5LCAibmJmIjogMTUxNDg1MTEzOSwgInJvbGUiOiAiYWRtaW4iLCAic3ViIjogIlltOWkifQ.5qsm7rRTvqFHAgiB6evX0a_hWnGbWquZC0HImVQPQo8"
```
Comment (anderseknert) [inline docs/content/http-api-authorization.md:307]: AFAICS, these tokens can't be reused here. They contain different claims than the ones used for the envoy tutorial.

**Review by anderseknert — COMMENTED** [review]
Thanks! Looks good, but let's use these new tokens only for the envoy tutorial.

File: docs/content/http-api-authorization.md (hunk 1 above)
Comment (rokkiter) [reply docs/content/http-api-authorization.md]: Thanks for your guidance! 
I have canceled this part of the code change.

**Review by anderseknert — APPROVED** [review]
Great! Thanks for contributing 👍

**Pull request merged**